pegasus/emails/volunteer_contact_2015_receipt.md

```diff
@@ -7,29 +7,49 @@ end
 %>
 to: '<%= format_email_address(volunteer_email_s, volunteer_name_s) %>'
 from: '"Tanya Parker (Code.org)" <tanya_parker@code.org>'
+reply-to: '<%= format_email_address(email_s, name_s) %>'
 subject: "A teacher is requesting your help for the Hour of Code"
 ---
 
 <% update_preferences = "http://#{CDO.canonical_hostname('code.org')}/volunteer/engineer/edit/#{volunteer_secret_s}/" %>
 
-### The following teacher is requesting your help for the Hour of Code
+Hi <%= volunteer_name_s %>,
 
-- **Name:** <%= name_s %>
-- **Email address:** [<%= email_s %>](<%= "mailto:" + email_s %>)
-- **School name:** <%= school_name_s %>
-- **School location:** <%= school_location_s %>
-- **Message:** <%= email_message_s %>
+<%= name_s %> is a teacher at <%= school_name_s %>. S/he found you on the Hour of Code site and after reviewing your profile specifically requested if you could help their class with the Hour of Code this year. S/he would like it if you could:
+
+***
+
+<% type_task_ss.each do |task| %>
+  <% if task == 'onsite' %>
```

Comment:
so even though a volunteer can volunteer for multiple roles, they just get picked for one per email?